src/lib/Select/Select.svelte

```diff
@@ -364,20 +362,6 @@
     --button-width: var(--multipleSelect-btn-width, 100%);
   }
 
-  .icon-container {
```

Comment:
why did we remove this code ?  
Container is added intentionally when we want to keep the images small & increase click-ability. Lets avoid removing intentionally added code.

Reply:
its dublicate